target/linux/ipq807x/Makefile

```diff
@@ -14,7 +14,7 @@ include $(INCLUDE_DIR)/target.mk
 DEFAULT_PACKAGES += \
 	kmod-usb3 kmod-usb-dwc3 kmod-usb-dwc3-qcom \
 	kmod-leds-gpio kmod-gpio-button-hotplug \
-	libwolfsslcpu-crypto \
+	libwolfsslcpu-crypto kmod-qca-nss-dp \
```

Comment:
Wonder if this can be dropped by first adding the qca-nss-dp package and then the target Makefile? Seems strange to first add the target that won't actually work as it won't have any ethernet driver and then add the package for it?

Reply:
Not really as we want NSS-DP and SSDK to depend on the target itself

Reply:
Oh right... Chicken egg problem... Then sorry for the noise.

Reply:
Yeah, same thing as with ath11k-ahb cause they are not universal so we end up with 2 commits anyway.